app/main/views/services.py

```diff
@@ -40,69 +33,58 @@ def list_services():
 @login_required
 @flask_featureflags.is_active_feature('EDIT_SERVICE_PAGE')
 def edit_service(service_id):
-    service = data_api_client.get_service(service_id).get('services')
+    service = data_api_client.get_service(service_id)
+    service_unavailability_information = service.get('serviceMadeUnavailableAuditEvent')
+    service = service.get('services')
 
     if not is_service_associated_with_supplier(service):
         abort(404)
 
-    return _update_service_status(service)
+    template_data = main.config['BASE_TEMPLATE_DATA']
+    framework = data_api_client.get_framework(service['frameworkSlug'])['frameworks']
 
+    content = content_loader.get_manifest(framework['slug'], 'edit_service').filter(service)
 
-@main.route('/services/<string:service_id>', methods=['POST'])
+    return render_template(
+        "services/service.html",
+        service_id=service.get('id'),
+        service_data=service,
+        service_unavailability_information=service_unavailability_information,
+        framework=framework,
+        sections=content.summary(service),
+        **dict(**template_data))
+
+
+@main.route('/services/<string:service_id>/remove', methods=['POST'])
 @login_required
 @flask_featureflags.is_active_feature('EDIT_SERVICE_PAGE')
-def update_service_status(service_id):
+def remove_service(service_id):
     service = data_api_client.get_service(service_id).get('services')
 
     if not is_service_associated_with_supplier(service):
         abort(404)
 
-    if not is_service_modifiable(service):
-        return _update_service_status(
-            service,
-            "Sorry, but this service isn't modifiable."
-        )
-
-    # Value should be either public or private
-    status = request.form.get('status', '').lower()
-
-    translate_frontend_to_api = {
-        'public': 'published',
-        'private': 'enabled'
-    }
+    # suppliers can't un-remove a service
+    if service.get('status') != 'published':
+        abort(400)
 
-    if status in translate_frontend_to_api.keys():
-        status = translate_frontend_to_api[status]
-    else:
-        return _update_service_status(
-            service,
-            "Sorry, but '{}' is not a valid status.".format(status)
-        )
-
-    try:
-        updated_service = data_api_client.update_service_status(
-            service.get('id'), status,
-            current_user.email_address)
+    updated_service = data_api_client.update_service_status(
+        service.get('id'),
+        'enabled',
```

Comment:
Changing status to `enabled` in order to disable the service... how did we get to this point again? :)

Might it be worth going all the way to `disabled` if we're treating both states the same, just for clarity?  Or do we still want to maintain the distinction between services turned off by an admin and services turned off by the supplier?